src/pages/Tournament/TournamentDetails/MatchManagementPanel/parts.tsx

```diff
@@ -0,0 +1,275 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Button, List, Modal } from 'antd';
+import { LaptopOutlined } from '@ant-design/icons';
+
+import { Row } from 'components/common';
+import {
+  MatchDetails,
+  MessageResponse,
+  Player,
+  PlayerStanding,
+  RoundDetails,
+} from 'utils/apiResponseShapes';
+import { ClosableComponentProps } from 'utils/interfaces';
+import useFetch from 'hooks/useFetch';
+import useToastPushSubmit from 'hooks/useToastPushSubmit';
+import useEventCallback from 'hooks/useEventCallback';
+import { PlayerColor, SpecialConditionScores } from 'utils/enums';
+import UpdateScoreDialog from './UpdateScoreDialog';
+import PlayerBox from './PlayerBox';
+import { useTournamentInfoContext } from '../TournamentInfoContext';
+import { useTournamentPlayerContext } from '../TournamentPlayerContext';
```

Comment:
Codacy has a fix for the issue: [Insert `';⏎import·PlayerBox·from·'./PlayerBox';⏎import·UpdateScoreDialog·from·'./UpdateScoreDialog`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import { useTournamentPlayerContext } from '../TournamentPlayerContext';
```